src/qpmodel.jl

```diff
@@ -1,31 +1,43 @@
+# override some NLPModels functions
+import NLPModels.jac_structure!, NLPModels.hess_structure!, NLPModels.jac_coord!, NLPModels.hess_coord!
+export jac_structure!, hess_structure!, jac_coord!, hess_coord!
+
 mutable struct QPData
-    c0::Float64                 # constant term in objective
-    c::AbstractVector{Float64}  # linear term
-    H::AbstractMatrix{Float64}  # quadratic term
-    opH::AbstractLinearOperator # assumed with preallocation!
-    A::AbstractMatrix{Float64}  # constraint matrix
+    c0  :: Float64                  # constant term in objective
+    c   :: AbstractVector{Float64}  # linear term
+    H   :: AbstractMatrix{Float64}  # quadratic term
+    opH :: AbstractLinearOperator   # assumed with preallocation!
+    A   :: AbstractMatrix{Float64}  # constraint matrix
```

Comment:
It's ok to add spaces around `::` but we don't usually align declarations this way.

Reply:
I'm happy with it tough ;-)